RELEASES.md

```diff
@@ -1,3 +1,193 @@
+Version 1.26.0 (2018-05-10)
+==========================
+
+Language
+--------
+- [Closures now implement `Copy + Clone` if all captured variables do.][49299]
+- [Stabilised the `!` type (A type that has no value).][47630]
+- [The inclusive range syntax e.g. `for x in 0..=10` is now stable.][47813]
+- [`impl Trait` is now stable allowing you to have abstract
+  return types.][49255] e.g. `fn foo() -> impl Iterator<Item=u8>`
```

Comment:
`impl Trait` can also appear in argument position e.g. `fn open(a: impl AsRef<Path>)`.